davidagitlen/Tripedia-fe
# Hooks form

#### What’s this PR do?
This PR will create a backbone for the UI with functional collapse form actions, as well as implement a template for how React Hooks will be utilized in this application.

#### Where should the reviewer start?
The reviewer should start in app and notice that all the components are created and their css is in the corresponding file. Then checkout the functionality of the FormContainer, and notice that hooks has been implemented in FormContainer and StartForm.

#### How should this be manually tested?
npm start and open the localhost, from there the tester should be able to click the button on the right of the FormContainer and see it collapse, clicking the button again will cause the FormContainer to reappear.  Then the tester should be able to to click the StartForm and see it collapse, when this happens the map should be flush with the FormContainer, if the StartForm collapses successfully it should turn yellow, show mock data, and remain at 8vh. If clicked again the StartForm should open back up and the inputs and button should be visible, at this point the map will not be flush at the bottom, the fix to this will be implemented tomorrow. 

#### Any background context you want to provide?
There are some hooks resources in the slack page for those interested in understanding what is happening with those particular lines of code. 

#### Questions:
 How do we want to move forward with the transition of the collapsing forms,  to smooth things out should we add animations? or does the action seem fine?


# Please squash this PR and entitle it "Front End Boilerplate"
